### ISSUE TITLE
questions about production server

### ISSUE THREAD

**Issue by efirvida** [stuartkuentzel/ng-wp-theme#13]
Hi i´m using uniserver as the web server so I create a vhos named **linda2.com** 

so my environment.prod.ts is:
```
export const environment = {
  production: true,
  wpBase: '//linda2.com/wp-json/wp/v2/'
}; 
```
and run the build command as 
`ng build --prod --base-href /wp-content/themes/wpng2/dist/`

and everithing works fine, but my home url on the browser change to http://linda2.com/wp-content/themes/wpng2/dist/ and I and want to ask, if its posible to make it work having the home url as: http://linda2.com

**Comment by stuartkuentzel** [comment]
yup, absolutely. Try running `ng build --prod --deploy-url="/wp-content/themes/{THEME_DIRECTORY_NAME}/dist/"`

**Comment by efirvida** [comment]
thanks :)

**Comment by akhilmekkatt** [comment]
How to set en variables local codebase 
`wpBase: '//localhost/wp-json/wp/v2/'`

is not giving result
